bundles/org.eclipse.equinox.bidi/src/org/eclipse/equinox/bidi/advanced/StructuredTextExpertFactory.java

```diff
@@ -21,38 +21,48 @@
 import org.eclipse.equinox.bidi.internal.StructuredTextImpl;
 
 /**
- * Obtains IStructuredTextExpert instances.
- * An {@link IStructuredTextExpert} instance (called in short an "expert") provides
- * the advanced methods to process a certain type of structured text, and 
- * is thus related to a specific 
- * {@link StructuredTextTypeHandler structured text type handler}.
- * There are two kinds of experts:
+ * Obtains IStructuredTextExpert instances. An {@link IStructuredTextExpert}
+ * instance (called in short an "expert") provides the advanced methods to
+ * process a certain type of structured text, and is thus related to a specific
+ * {@link StructuredTextTypeHandler structured text type handler}. There are two
+ * kinds of experts:
  * <ul>
- *   <li>stateful, obtained by calling {@link #getStatefulExpert}.</li>
- *   <li>not stateful, obtained by calling {@link #getExpert}.</li>
- * </ul>  
- * <p>Only the stateful kind can remember the state established by a call to
- * a text processing method and transmit it as initial state in the next call
- * to a text processing method.
- * <p>In other words, the methods 
- * {@link IStructuredTextExpert#getState()},
+ * <li>stateful, obtained by calling {@link #getStatefulExpert}.</li>
+ * <li>not stateful, obtained by calling {@link #getExpert}.</li>
+ * </ul>
+ * <p>
+ * Only the stateful kind can remember the state established by a call to a text
+ * processing method and transmit it as initial state in the next call to a text
+ * processing method.
+ * <p>
+ * In other words, the methods {@link IStructuredTextExpert#getState()},
  * {@link IStructuredTextExpert#setState} and
- * {@link IStructuredTextExpert#clearState()} of 
- * {@link IStructuredTextExpert} are inoperative for experts which are not stateful.
+ * {@link IStructuredTextExpert#clearState()} of {@link IStructuredTextExpert}
+ * are inoperative for experts which are not stateful.
  * <p>
- * Using a stateful expert is more resource intensive, thus not stateful
- * experts should be used when feasible. 
+ * Using a stateful expert is more resource intensive, thus not stateful experts
+ * should be used when feasible.
  */
 final public class StructuredTextExpertFactory {
 
 	/**
-	 * The default set of separators used to segment a string: dot, colon, slash, backslash.
+	 * The default set of separators used to segment a string: dot, colon, slash,
+	 * backslash.
 	 */
 	private static final String defaultSeparators = StructuredTextProcessor.getDefaultSeparators();
 
 	static private Map<String, IStructuredTextExpert> sharedDefaultExperts = new HashMap<>(); // String type -> expert
 
-	static private Map<String, Map<StructuredTextEnvironment, IStructuredTextExpert>> sharedExperts = new HashMap<>(); // String type -> map of { environment -> expert }
+	static private Map<String, Map<StructuredTextEnvironment, IStructuredTextExpert>> sharedExperts = new HashMap<>(); // String
+																														// type
```

Comment:
This comment looks pretty weird.

Reply:
Should I move this to the line above instead or should I remove it?
I'm not sure if actually adds any value as it's only a repeat of what data types that are used.

Reply:
Just delete it.